client/signup/steps/woocommerce-install/confirm/index.tsx

```diff
@@ -63,8 +63,18 @@ export default function Confirm( props: WooCommerceInstallProps ): ReactElement
 		siteUpgrading,
 		hasBlockers,
 		warnings,
+		isReadyForTransfer,
 	} = useWooCommerceOnPlansEligibility( siteId );
 
+	// Skip to transfer step if the site is ready for transfer.
+	useEffect( () => {
+		if ( ! isReadyForTransfer ) {
+			return;
+		}
+
+		goToStep( 'transfer' );
```

Comment:
Real nitpick, but would this be clearer as...?

```suggestion
		if ( isReadyForTransfer ) {
			goToStep( 'transfer' );
		}
```

Reply:
👍